models/clm/CAM_bias_correction/CAM6_site_grid.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Extracts site level grid cell from CAM6 reanalysis
+
+# CAM6 Reanalysis
+
+CAM6_grid_path="/glade/collections/rda/data/ds345.0/cpl_unzipped/"
+# Site level  grid cell from CAM6 reanalysis
+CAM6_site_path="/glade/work/bmraczka/CAM6_NR1/"
+
+# The US-NR1 flux tower site is used as an example 
+# NR1 location (40.03, -105.55) or (40.03, 254.45)
+# CAM6 grid is 1.25x0.95  or 288 longitude grids and 192 latitude grids
+# This lat/lon corresponds with doma_lon= 204 (255 degrees east) ; doma_lat= 138  ; (40.0524 degrees)
```

Comment:
Defining the lat and lon points here is even more useful than in CAM4_site_grid.sh.